modal.js

```diff
@@ -0,0 +1,41 @@
+const popup = document.getElementById('myPopup');
```

Comment:
Unfortunately, your first See Project button opens the popup but other projects do not. Also, icons on the popup window look broken. These two problems were not there before the last review. They introduced after that 😞  Kindly fix the issues and make sure they work as before with your new refactored code 👍 
![lala](https://user-images.githubusercontent.com/25887253/231568473-a7064310-20e9-4e3a-9b4f-86e9540c9566.gif)